connector/src/main/java/org/geysermc/connector/network/ConnectorServerEventHandler.java

```diff
@@ -95,6 +95,17 @@ public BedrockPong onQuery(InetSocketAddress inetSocketAddress) {
             pong.setMaximumPlayerCount(config.getMaxPlayers());
         }
 
+        // The ping will not appear if the MOTD + sub-MOTD is of a certain length.
+        if (pong.getMotd().length() + pong.getSubMotd().length() > 339) {
+            // Split the difference
+            if (pong.getMotd().length() > 170) {
+                pong.setMotd(pong.getMotd().substring(0, 170));
+            }
+            if (pong.getSubMotd().length() > 170) {
+                pong.setSubMotd(pong.getSubMotd().substring(0, 170));
+            }
```

Comment:
Can this be changed to allow for one being slightly longer and one being slightly shorter than 170 characters?
EG:
motd: `Hello, World!` (13)
submotd: `Some super long second motd line, with extra text! Damn didn't realise this had to be this long to hit the character limit. This is going on for a while and my brain is running out of things to type into this.` (209)
Total of 222 which is less than 339 so should not cause an issue with the packet.

Reply:
That code won't be hit unless the total length is greater than 339 (see the if statement right above).